### PR TITLE
CLIENTS-687: Update dropdowns to match original designs

### PR DESCRIPTION
**Pull request by tomasstark**
This PR addresses [CLIENTS-687](https://laterpay.atlassian.net/browse/CLIENTS-687).

- Updates styling of filter dropdowns to match the original designs.

- Updates styling of autocomplete options to match the original designs.

- Updates styling of some surrounding elements to match the designs too.

- Fixes spacing and font sizes on dashboard screens to match designs.

### CODE REVIEW

File: assets/src/js/pages/revenue-generator-contribution.js [hunk 1]
```diff
@@ -268,7 +289,9 @@ import { __, sprintf } from '@wordpress/i18n';
 				} );
 
 				// Copy Contribution code on Dashboard.
-				$o.contributionDashboardShortcodeLink.on( 'click', function( e ) {
+				$o.contributionDashboardShortcodeLink.on( 'click', function(
+					e
```
Comment (fgenghini) [inline assets/src/js/pages/revenue-generator-contribution.js:293]: I see you have "reformatted" the code in many places and it is great. However, in some parts (like this one) the new format looks "weird" (I would probably keep the old format here with everything in one line). Question: are you using any tool (linter) that (re)formats the code automatically?

Comment (tomasstark) [reply assets/src/js/pages/revenue-generator-contribution.js:293]: This is the format that rtCamp originally introduced and the code is automatically reformatted when running an npm build task. The code in red is what I wrote, the code in green is what linter produced. I agree this is not readable at all and I'd adjust linter to stop creating these huge unnecessary gaps everywhere. I'll go and create a ticket in Jira for that. 👍

Comment (tomasstark) [reply assets/src/js/pages/revenue-generator-contribution.js:293]: Created [here](https://laterpay.atlassian.net/browse/CLIENTS-768).

**Review by fgenghini — APPROVED** [review]
🚀

**Pull request merged**